common/src/main/java/com/wynntils/services/itemfilter/ItemFilter.java

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.services.itemfilter;
```

Comment:
Can you move the interfaces to a new, `type` sub package? And the implementations in `filters`.